financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupViewModel.kt

```diff
@@ -91,7 +94,12 @@ internal class NetworkingLinkLoginWarmupViewModel @AssistedInject constructor(
         suspend {
             eventTracker.track(Click("click.continue", PANE))
             // Trigger a lookup call to ensure we cache a consumer session for posterior verification.
-            lookupAccount(payload.email)
+            lookupAccount(
+                email = payload.email,
+                emailSource = EmailSource.CUSTOMER_OBJECT,
```

Comment:
@tillh-stripe now passing the email source on the new lookup call in warmup (hardcoded to customer_object since it cannot be manually edited)